css/_polls.scss

```diff
@@ -0,0 +1,62 @@
+.pollCreateFormContainer, .voteFormContainer, .pollResultsFormContainer {
+    width: 500px;
```

Comment:
Width should already be taken care of by the modal. As such I don't know if it's necessary to set a determined width.